### ISSUE TITLE
[Analytics Hub] Configure Revenue Analytics card

### ISSUE THREAD

**Issue by ThomazFB** [woocommerce/woocommerce-ios#8150]
Depends on #8148

## Summary
The Revenue analytics card should be able to display two types of information: `Total sales` and `Net sales`. Both can be properly acquired directly from the Woo Core stats API.

We will integrate real data from the API in #8189. Until then, we can use fake data for the previous and current period, and generate a percentage delta value based on the Total sales and Net sales difference between the two periods.

## Designs

![Image](https://user-images.githubusercontent.com/5920403/202560240-e304ff19-35b7-4937-86bb-a1d240793712.png)

**Comment by rachelmcr** [comment]
Work done so far:

* Added methods to `StatsDataTextFormatter` to retrieve and format the `Total Sales` and `Net Sales` amounts and percentage delta values.
* Updated `AnalyticsHubViewModel` to show placeholder values and localized strings on the Revenue card.

To be done:

I started to work on configuring the Revenue card with `OrderStatsV4` data, but given the latest discussion about the data flow implementation (internal ref: p1669291372706539-slack-C02KUCFCSFP) I think this should be put on hold for now. Once that implementation is done we'll have a better idea of what remaining work (if any) is needed to update the Revenue card with real data from the API.

**Comment by rachelmcr** [comment]
Closing this issue since the data binding for this card was handled in #8229.
